src/Integration/Resources/Strings.resx

```diff
@@ -314,8 +314,8 @@ ocalization note: '[try again]()' is a special markup used to translate the text
     <comment>Server context menu command display text to show all projects (inc. unbound projects)</comment>
   </data>
   <data name="DownloadingQualityProfileProgressMessage" xml:space="preserve">
-    <value>Download quality profile(s):</value>
```

Comment:
The old message appeared once before the first QP was downloaded, then replaced with a blank message. Now each QP is listed as it is being downloaded.